src/_langs/en/updates/2015-07-22-access-ble-devices-with-web-bluetooth.markdown

```diff
@@ -0,0 +1,279 @@
+---
+
+layout: update
+published: true
+
+collection: updates
+category: chrome
+product: chrome
+type: news
+date: 2015-07-22
+
+title: "Access BLE devices with Web Bluetooth"
+description: "A Web API has been added to Chrome OS M45 that
+makes it possible for websites to discover and communicate with devices over
+the Bluetooth 4 wireless standard using GATT."
+article:
+  written_on: 2015-07-22
+  updated_on: 2015-07-22
+
+authors:
+  - beaufortfrancois
+tags:
+  - Bluetooth
+  - IoT
+  - WebBluetooth
+  - PhysicalWeb
+permalink: /updates/2015/07/access-ble-devices-with-web-bluetooth.html
+---
+
+How about I tell you websites could communicate with nearby Bluetooth devices
+in a secure and privacy-preserving way? This way, heart rate monitors, singing
+lightbulbs and [turtles](https://www.youtube.com/watch?v=1LV1Fk5ZXwA) could
+be managed directly from a website.
+
+## Before we start
+
+This article assumes you have some basic knowledge of how works Bluetooth Low
+Energy (BLE) and Generic Attribute Profile (GATT).
+
+Even though the [Web Bluetooth API
+specification](https://webbluetoothcg.github.io/web-bluetooth/) is not
+finalized yet, the Chrome Team is actively looking for enthusiastic developers
+(I mean you) to try out this work-in-progress API and give
+[feedback](https://code.google.com/p/chromium/issues/entry?labels=Cr-Blink-Bluetooth).
+
+Web Bluetooth API is at the time of writing partially implemented in Chrome OS
+M45 behind an experimental flag. Go to `chrome://flags/#enable-web-bluetooth`,
+enable the highlighted flag, restart Chrome and you should be able to
+[scan](#scan-bluetooth-devices) and [connect](#connect-to-a-bluetooth-device)
+to nearby Bluetooth devices and
+[read](#read-a-bluetooth-characteristic)/[write](#write-to-a-bluetooth-characteristic)
+Bluetooth characteristics.
+
+<img src="/web/updates/images/2015-07-22-access-ble-devices-with-web-bluetooth/web-bluetooth-flag.png" alt="Web Bluetooth Flag highlighted in chrome://flags"/>
+
+## Security Requirements
+
+### HTTPS Only
+
+Because the Web Bluetooth API is a powerful new feature added to the Web,
+Google Chrome aims to make it [available only to secure
+origins](https://www.chromium.org/Home/chromium-security/prefer-secure-origins-for-powerful-new-features).
```

Comment:
Maybe:

`Because the Web Bluetooth API is a powerful new feature added to the Web, Google Chrome aims to make it available only to [secure contexts](https://w3c.github.io/webappsec/specs/powerfulfeatures/#intro).`

The chromium.org page was just the proposal for Powerful Features spec. That got split into a TAG document that hasn't been published yet and will define what features should be limited, and the Secure Contexts spec that defines how to limit them.